### ISSUE TITLE
Unknown purpose

### ISSUE THREAD

**Issue by Tobychev** [Fortran-FOSS-Programmers/Fortran-FOSS-Programmers.github.io#1]
Hello!

I like the implications of the title of the repo, but maybe you could put in a readme with at least a sentence describing what you hope fortran-foss-programmers will become?

kind regards
/Tobychev

**Comment by zbeekman** [comment]
yes, I agree…. I’ve been a bit busy recently but will try to get at least a brief mission statement/outline up later today or tomorrow.

**Comment by Tobychev** [comment]
Nice! :)
